### ISSUE TITLE
Button-element cannot not use Size and IsFullWidth at the same time

### ISSUE THREAD

**Issue by viktorvan** [MangelMaxime/Fulma#155]
I tried to create a large fullwidth button:

```
Button.button 
    [ Button.IsFullWidth
    ; Button.Size IsLarge
    ; Button.Color IsInfo
    ] 
    [ str "Text" ] 
```
but it seems you cannot set the Size And IsFullWidth at the same time since they both will set the size property when parsing  the options.

https://github.com/MangelMaxime/Fulma/blob/fcfcdf9da9f0a937801e7ccdb0d1895905ca98b7/src/Fulma/Elements/Button.fs#L110
https://github.com/MangelMaxime/Fulma/blob/fcfcdf9da9f0a937801e7ccdb0d1895905ca98b7/src/Fulma/Elements/Button.fs#L112

Is this by design? I can workaround it by using a custom class for either one of course, but I was just a bit confused as to what was happening until I looked at the source code.

**Comment by MangelMaxime** [comment]
I think at the time I was thinking we couldn't use both at the same time. But looking at Bulma source code I guess both can be used together they don't set the same properties.

**Comment by MangelMaxime** [comment]
Released in `Fulma 1.0.0-beta-024`
